doc/source/style.ipynb

```diff
@@ -62,9 +61,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
```

Comment:
Not a big deal, I think I have another PR in the works that does the same thing.